test/resources/Documents.test.ts

```diff
@@ -4,21 +4,33 @@ import { Applicant, Document, OnfidoDownload } from "onfido-node";
 import { createNock, onfido, getExpectedObject, createApplicant, cleanUpApplicants, uploadDocument } from "../testHelpers";
 import { exampleDocument } from "../testExamples";
 
-function getExpectedDocument(exampleDocument: Document)
+function getExpectedDocument(exampleDocument: Document, overrideProperties={})
 {
   return getExpectedObject(exampleDocument, {
     'applicantId': applicant.id,
-    'downloadHref': expect.stringMatching(/^\/.+/) });
+    'downloadHref': expect.stringMatching(/^\/.+/),
+    ... overrideProperties });
+}
+
+function sortByType( a: Document, b: Document ) {
+  if ( a.type < b.type ){
+    return -1;
+  }
+  if ( a.type > b.type ){
+    return 1;
+  }
+  return 0;
 }
```

Comment:
I believe these helper methods should be either at the end of the spec file instead or on a dedicated helper file. Not a strong opinion and therefore will leave that up to you 💅

Reply:
Unfortunately we can't factorize them in a shared file as they act on different parameters. Nevertheless I agree it's still worth it to but them aside even without that.